.gitignore

```diff
@@ -123,6 +123,14 @@ cluster_scripts/log_files
 *.stderr
 *.out
 *-log.txt
+model_training/*
+wandb/*
+# run scripts
+run_*.sh
+
+# configs and data structures
+*_config.json
+*_ds.pkl
```

Comment:
not sure that these need to be in the repo `.gitignore` file, but I guess there isn't really any harm in having extra stuff here (unless there is @hmacdope?)

Reply:
No harm, but you shouldn't be in the habit of generating anything inside the source tree anyway.

Reply:
stripped out the generated file strings